base/applications/rapps/lang/bg-BG.rc

```diff
@@ -74,17 +74,17 @@ BEGIN
     PUSHBUTTON "Из&бор", IDC_CHOOSE, 187, 85, 50, 14
     AUTOCHECKBOX "След слагане, използваните файлове да се изтриват", IDC_DEL_AFTER_INSTALL, 16, 100, 218, 12
     GROUPBOX "Software source", -1, 4, 118, 244, 46
-    CONTROL "Use default", IDC_SOURCE_DEFAULT, "Button", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 15, 132, 74, 10
-    CONTROL "Specified source", IDC_USE_SOURCE, "Button", BS_AUTORADIOBUTTON | WS_TABSTOP, 15, 147, 74, 10
+    AUTORADIOBUTTON "Use default", IDC_SOURCE_DEFAULT, 15, 132, 74, 10, BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP
```

Comment:
If you use `AUTORADIOBUTTON`,  the `BS_AUTORADIOBUTTON` is useless.

Reply:
No that is not true. If I specify at least one additional EX_STYLE, then I do have to specificy them all. See https://learn.microsoft.com/de-de/windows/win32/menurc/autoradiobutton-control

Reply:
See also the matches in the notepad module https://git.reactos.org/?p=reactos.git&a=search&h=c3a45b679dc4b0452eab260b8e573a8360be37aa&st=grep&s=+AUTORADIOBUTTON

Reply:
I tested your suggestion on Windows and it seems as if you are correct. It works even without that. They do still stay mutually exclusive. I will strip it everywhere. Seems to contradict the MS documentation a bit for me though.